### PR TITLE
[CBRD-23690] Improvement of 'Rebuilding Replication Script'

### PR DESCRIPTION
**Pull request by JoohoK**
http://jira.cubrid.org/browse/CBRD-23690
Added 'ssh_port' option to support other ports when using ssh or scp utility. 
There are two cases how ssh/scp is used in the script(ha_make_slavedb.sh)
1. ssh or scp is used directly 
* add a port option to ssh/scp command
2. ssh/scp utility is used through the functions in other file (scp_from.exp, scp_to.exp, ssh.exp)
* pass the ssh_port as a argument

### CODE REVIEW

**Review by mhoh3963 — APPROVED** [review]

File: contrib/scripts/ha/README [hunk 1]
```diff
@@ -16,7 +16,7 @@ Before you run ha_make_slavedb.sh, you should configure some parameters in ha_ma
   - backup_option    : Additional options for cubrid backupdb. Optional.
   - restore_option   : Additional options for cubrid restoredb. Optional.
   - scp_option       : scp utility options. ha_make_slavedb.sh uses 'scp' to copy a backup volume from a master to a slave. The default values of '-l 16M' means there's a network limitation of 16MB.
-
+  - ssh_port         : port number for scp and ssh utility.
```
Comment (wrlawodms) [inline contrib/scripts/ha/README:19]: All the other options are classified into three categories in `example)` below, which is mandatory, conditional and optional. How about adding `ssh_port`?

File: contrib/scripts/ha/expect/scp_from.exp [hunk 2]
```diff
@@ -5,11 +5,12 @@ set passwd [lindex $argv 1]
...
-spawn scp -r $user@$host:$source $target
+spawn scp -P $port -r $user@$host:$source $target
 expect {
   "(yes/no)?" {send "yes\r";exp_continue}
   "password:" {send "$passwd\r"}
 }
 sleep 2
-expect eof
\ No newline at end of file
+expect eof
```
Comment (wrlawodms) [inline contrib/scripts/ha/expect/scp_from.exp:16]: Is this change on purpose?

File: contrib/scripts/ha/ha_make_slavedb.sh [hunk 3]
```diff
@@ -238,7 +239,7 @@ function get_output_from_replica()
 	mkdir $output
 	
 	for replica_host in ${replica_hosts[@]}; do
-		scp_from_expect "$cubrid_user" "$server_password" $output $replica_host $output/$replica_host 
+		scp_from_expect "$cubrid_user" "$server_password" $output $replica_host $output/$replica_host
```
Comment (wrlawodms) [inline contrib/scripts/ha/ha_make_slavedb.sh:242]: an unrelated change?

File: contrib/scripts/ha/README (hunk 1 above)
Comment (JoohoK) [reply contrib/scripts/ha/README:19]: yes, it is better to add ssh_port to below example. I will fix

File: contrib/scripts/ha/expect/scp_from.exp (hunk 2 above)
Comment (JoohoK) [reply contrib/scripts/ha/expect/scp_from.exp:16]: No, not on purpose.

File: contrib/scripts/ha/ha_make_slavedb.sh (hunk 3 above)
Comment (JoohoK) [reply contrib/scripts/ha/ha_make_slavedb.sh:242]: yes, unrelated change. once I modified this line but reverted it to the original.

Comment (wrlawodms) [reply contrib/scripts/ha/ha_make_slavedb.sh:242]: Why does it say that it has been changed although you reverted it?

**Review by hornetmj — APPROVED** [review]

File: contrib/scripts/ha/ha_make_slavedb.sh (hunk 3 above)
Comment (JoohoK) [reply contrib/scripts/ha/ha_make_slavedb.sh:242]: it shows the change because the space after "$output/$replica_host" is deleted unintendly.

Comment (wrlawodms) [reply contrib/scripts/ha/ha_make_slavedb.sh:242]: Ok, I misunderstood your comment. I thought you reverted unintended change already.

Comment (wrlawodms) [reply contrib/scripts/ha/ha_make_slavedb.sh:242]: So, please remove all the unintended changes.

Comment (JoohoK) [reply contrib/scripts/ha/ha_make_slavedb.sh]: OK, I removed all the unintended changes

**Review by wrlawodms — APPROVED** [review]
LGTM
